test/test_torch.py

```diff
@@ -7623,6 +7657,16 @@ def test_rot90(self, device):
         self.assertRaises(RuntimeError, lambda: data.rot90(1, [0, 1, 2]))
         self.assertRaises(RuntimeError, lambda: data.rot90(1, [0]))
 
+    @dtypes(torch.cfloat, torch.cdouble)
+    @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
+    def test_complex_rot90(self, device, dtype):
```

Comment:
Same question as above: is there an existing test for rot90 and/or should this test be expanded with additional dtypes?